test/unit/server.test.ts

```diff
@@ -220,5 +220,24 @@ describe('server client', function () {
         done();
       }, 20);
     });
+
+    // eslint-disable-next-line
+    it('retrieves context from the same request object', (done) => {
+      let request1 = {};
+      let request2 = {};
```

Comment:
This is not used. Did you mean to use it?

Reply:
No, probably forgot. Good catch!